app/templates/components/preprint-footer-branded.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="text-center">
+    <span>{{model.name}}:</span>
+    <div style="display: inline;">
+        <span>
+            <a href="mailto:{{if model.emailSupport model.emailSupport (concat 'contact+' model.id '@osf.io')}}"
+               onbeforeclick={{action 'click' 'link' 'Branded Footer - Support'}}
+            >
+                {{t "components.preprint-footer-branded.support"}}
+            </a>
+        </span>
+        <span> | </span>
+        <span>
+            <a href="mailto:{{if model.emailContact model.emailContact (concat 'support+' model.id '@osf.io')}}"
```

Comment:
Nice catch, I also forgot to i18n-ize the aria-labels as well. Will change both.